simple_graph/weighted_graph.py

```diff
@@ -0,0 +1,170 @@
+#! /usr/bin/env python
+
+# Simple Graph project for Efrain, Henry, Mark
+# Graph is unweighted but directed
+
+import time
+from collections import OrderedDict
+
+
+def timed_func(func):
+    """Decorator for timing our traversal methods."""
+    def timed(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        elapsed = time.time() - start
+        print "time expired: %s" % elapsed
+        return result
+    return timed
+
+
+class Wgraph(object):
+    """A class defining a unweighted direct simple graph."""
+
+    def __init__(self):
+        """Instantiates simple graph"""
+        self.dict = {}
+
+    def nodes(self):
+        """return list of all nodes in a graph"""
+        return self.dict.keys()
+
+    def edges(self):
+        """return a list of all edges in the graph"""
```

Comment:
You might take the step of noting the internal data structure of the edges you return.  Are they two-tuples of `(start, end)` or three-tuples of `(start, end, weight)` or what?